### PR TITLE
Free-courses-en.md updated

### PR DESCRIPTION
**Pull request by dvig14**
## What does this PR do?
 Add resource(s)

### Description
This is Html and CSS , Bootstrap beginner-friendly course.

### Why is this valuable (or not)?
 This helps learners to clearly understand the concepts of HTML , CSS (media queries ,flex box ,grid)..

### How do we know it's really free?
This course is available on YouTube and free for all.

### For book lists, is it a book? For course lists, is it a course? etc.
It is a course.

## Checklist:
- [x] Read our [contributing guidelines](https://github.com/EbookFoundation/free-programming-books/blob/main/docs/CONTRIBUTING.md).
- [x] [Search](https://ebookfoundation.github.io/free-programming-books-search/) for duplicates.
- [x] Include author(s) and platform where appropriate.
- [x] Put lists in alphabetical order, correct spacing.
- [x] Add needed indications (PDF, access notes, under construction).
- [x] Used an informative name for this pull request.

## Follow-up

- Check the status of GitHub Actions and resolve any reported warnings!

### CODE REVIEW

File: courses/free-courses-en.md [hunk 1]
```diff
@@ -618,11 +619,11 @@
 * [Web Development By Doing: HTML / CSS From Scratch](https://www.udemy.com/course/web-development-learn-by-doing-html5-css3-from-scratch-introductory/) - Udemy
 * [What the Flexbox](https://flexbox.io) - Wesbos
 
-
```
Comment (bugbunnyy) [inline courses/free-courses-en.md:621]: Don't delete this line, there should be a blank lines between last section and next heading.

File: courses/free-courses-en.md [hunk 2]
```diff
@@ -632,7 +633,6 @@
 * [Learn Bootstrap 5 and SASS by Building a Portfolio Website - Full Course](https://www.youtube.com/watch?v=iJKCj8uAHz8) - Patrick Muriungi, freeCodeCamp (YouTube)
 * [Rapid website design with Bootstrap](https://www.udemy.com/course/responsive-website-design) - Laurence Svekis (Udemy)
 
-
```
Comment (bugbunnyy) [inline courses/free-courses-en.md:635]: Here as well

Comment (dvig14) [reply courses/free-courses-en.md]: Is it Ok now..

Comment (bugbunnyy) [reply courses/free-courses-en.md]: yup

File: courses/free-courses-en.md [hunk 3]
```diff
@@ -603,6 +603,7 @@
 * [Get to know HTML Learn HTML Basics](https://www.udemy.com/course/html-online-course/) - Laurence Svekis (Udemy)
 * [HTML & CSS](https://www.youtube.com/playlist?list=PLillGF-RfqbZTASqIqdvm1R5mLrQq79CU) - Brad Traversy, Traversy Media (YouTube)
 * [HTML & CSS Crash Course](https://www.youtube.com/playlist?list=PL4cUxeGkcC9ivBf_eKCPIAYXWzLlPAm6G) - The Net Ninja (Shaun Pelling)
+* [HTML & CSS Crash Course Tutorial For Beginners](https://youtube.com/playlist?list=PLr6-GrHUlVf_HUfo4LyFepGmj23nwqNdk) - EJ Media (YouTube)
```
Comment (shrn01) [inline courses/free-courses-en.md:606]:
```suggestion
* [HTML & CSS Crash Course Tutorial For Beginners](https://youtube.com/playlist?list=PLr6-GrHUlVf_HUfo4LyFepGmj23nwqNdk) - EJ Media
```

File: courses/free-courses-en.md [hunk 4]
```diff
@@ -623,6 +624,7 @@
 
 * [Bootstrap 4 Quick Start: Code Modern Responsive Websites](https://www.udemy.com/course/bootstrap-4) - Brad Hussey (Udemy)
 * [Bootstrap 5 Tutorial for Beginners](https://www.youtube.com/playlist?list=PL4cUxeGkcC9joIM91nLzd_qaH_AimmdAR) - The Net Ninja
+* [Bootstrap Tutorial for Beginners](https://youtube.com/playlist?list=PLr6-GrHUlVf-gjvHuzCnVmeuaeAjRGltv) - EJ Media (YouTube)
```
Comment (shrn01) [inline courses/free-courses-en.md:627]:
```suggestion
* [Bootstrap Tutorial for Beginners](https://youtube.com/playlist?list=PLr6-GrHUlVf-gjvHuzCnVmeuaeAjRGltv) - EJ Media
```

**Comment by eshellman** [comment]
Thanks, and welcome to free-programming-books!

**Pull request merged**